honeycomb-core/src/cells/identifiers.rs

```diff
@@ -0,0 +1,42 @@
+//! Module short description
```

Comment:
Not sure about having a separate file for that.
These ids are mostly used as internal indices, so keeping them close to data structures also make sense.

Reply:
I have grouped them here at the moment since I will be re-implementing data structures for attributes sooner than expected (see #30)